react/features/authentication/middleware.js

```diff
@@ -114,7 +115,16 @@ MiddlewareRegistry.register(store => next => action => {
                 && error.name === JitsiConnectionErrors.PASSWORD_REQUIRED
                 && typeof error.recoverable === 'undefined') {
             error.recoverable = true;
-            store.dispatch(_openLoginDialog());
+
+            // SONRISA MODIFICATION: instead of show login form, login user with credentials set by native app
```

Comment:
Again, I don't think it's necessary to state that this is a modification by sonrisa